content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -131,6 +131,25 @@ iptables:
 ...
 ```
 
+##### Performance improvements for `iptables` mode {#minimize-iptables-restore}
```

Comment:
```suggestion
##### Performance optimization for `iptables` mode {#minimize-iptables-restore}
```
(describe the state, not the diff - see https://developers.google.com/style/timeless-documentation for a good explanation on the motivation)

Now it's not an experiment, we should reframe.